WalletWasabi.Documentation/Guides/UpgradeTorDaemons.md

```diff
@@ -0,0 +1,31 @@
+Check if there is a new Tor client. 
+
+Tor changelog here: https://gitweb.torproject.org/tor.git/plain/ChangeLog
+
+Download the latest stable Tor Browser from here: https://www.torproject.org/download/languages/
+
+- Windows x32
+- Linux x64
+- macOS x64
+
+Do not copy PluggableTransports folder!
+
+## Windows
+msi => Browser\TorBrowser\Tor
+
+## Linux
+tar.gz => tor-browser_en-US\Browser\TorBrowser\Tor
+
+## macOS
+dmg => Tor Browser.app\Contents\MacOS\Tor
+
+Do not delete Tor file from the original folder!
+
+## Geoip files
+
+msi => Browser\TorBrowser\Data\Tor
+
+## Digest creation 
+
+WalletWasabi\WalletWasabi.Packager> dotnet run -- onlycreatedigests
+Copy hashes without file names into WalletWasabi\WalletWasabi\TorDaemons\digests.txt (order of the hashes doesn't matter)
```

Comment:
What hash function is used?

Reply:
Who cares? 😄

Reply:
I do. :-) I prefer clarity.

Reply:
SHA256 https://github.com/molnard/WalletWasabi/blob/d9ef511de936c649b275844588f39eff8a593aa1/WalletWasabi/Helpers/HashHelpers.cs#L19

Reply:
Yeah. It would be nice to have: `Digest creation (SHA-256)`.